Tests/CoreAPITests/WordPressOrgXMLRPCApiTests.swift

```diff
@@ -128,15 +119,10 @@ class WordPressOrgXMLRPCApiTests: XCTestCase {
             failure: { (error, _) in
                 expect.fulfill()
 
-                XCTAssertTrue(error is WordPressOrgXMLRPCApiError)
```

Comment:
This is the only change in behavior from the rewrite.

The `NSError`-converted error (via `asNSerror()`) that the API passes to this failure branch doesn't convert back to `WordPressOrgXMLRPCApiError`.

Reply:
I might have missed something. Does that mean the `error` in the failure block was `WordPressOrgXMLRPCApiError`, but in this PR it's changed to something else?